src/Game.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Game() {
+    return (
+     <h1>THIS IS GAME.js!!!</h1>
+  );
+}
```

Comment:
same deal as above, you could change this to:

```suggestion
const Game = () => (
     <h1>THIS IS GAME.js!!!</h1>
  );
```